### DISCUSSION TITLE
Quarto CLI for Windows installed, but WSL could not find the path.

### DISCUSSION THREAD

**Discussion by ariewjy** [quarto-dev/quarto-cli#6238] [Q&A] [answered]
### Description

Hi, 

I have installed quarto in my Windows machine, however it is accessible only in windows powershell or cmd, but not in WSL as you can see. 

![image](https://github.com/quarto-dev/quarto-cli/assets/73975333/fb524eac-4029-4007-a209-d3a598379626)

In VScode, it seems like the WSL could not find the path to the right quarto executable. 
![image](https://github.com/quarto-dev/quarto-cli/assets/73975333/da54effb-ed66-4f75-860f-97bae2c9eb9d)

but in Git Bash, it worked! Not sure why
![image](https://github.com/quarto-dev/quarto-cli/assets/73975333/3491413b-f74f-4c4d-bdd9-6862f030419a)

Does anyone have the same issue before?

Appreciate the help!

**Comment by cderv** [comment, marked as answer]
WSL (Windows Subsystem for Linux) is running Linux.  You need to install a Linux version of the tool you use in there, according to the OS you are using. So install Quarto for the Linux distribution you are using, otherwise it won't be found as you are observing. This will be the case for all the tool on your system. WSL is Linux, not Windows.  

> but in Git Bash, it worked! Not sure why

You can't use a windows tools directly inside WSL because it runs Linux. Git bash is just a Terminal which emulates a bash terminal, but you are running Windows tool in there (or Windows port to Linux tool like `grep` or other command).

**Reply by ariewjy** [reply]
Chris, thanks for your time explaining this as simple as possible! This makes sense, and I got it working by installing it via ubuntu wsl directly.

Thanks again!
